### PR TITLE
Nav component

### PR DESCRIPTION
**Pull request by mdvb1001**
notice and alert in one component

### CODE REVIEW

File: app/assets/javascripts/components/alert.es6.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    return (
+      <div className={`alert alert-${ this.props.alert_class }` +
+      ` alert-dismissible fade show`} role="alert">
+        <button type="button"
+                className="close"
+                data-dismiss="alert"
+                aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+          {this.props.text }
```
Comment (cbrenner04) [inline app/assets/javascripts/components/alert.es6.jsx]: this should be aligned with `<button>` and add a space before `this`

File: app/assets/javascripts/components/alert.es6.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class Alert extends React.Component{
+  render() {
+    return (
+      <div className={`alert alert-${ this.props.alert_class }` +
+      ` alert-dismissible fade show`} role="alert">
```
Comment (cbrenner04) [inline app/assets/javascripts/components/alert.es6.jsx]: align the template literals please

Comment (mdvb1001) [inline app/assets/javascripts/components/alert.es6.jsx]: ok check it out

File: app/assets/javascripts/components/alert.es6.jsx (hunk 1 above)
Comment (mdvb1001) [inline app/assets/javascripts/components/alert.es6.jsx]: Ok

**Pull request merged**